src/en/models-config.md

```diff
@@ -155,27 +162,25 @@ upgrades in order to provision machines faster.
 
 Two Boolean configuration options are available to disable APT updates and
 upgrades: `enable-os-refresh-update` (apt-get update) and `enable-os-upgrade`
-(apt-get upgrade), respectively. By default, these are both set to 'true'.
+(apt-get upgrade), respectively.
 
```

Comment:
Because it was already stated:

"...its default behaviour is to upgrade existing packages to their latest version"